src/checkPublication.ts

```diff
@@ -176,7 +176,7 @@ export function checkPublication(
     allLogs: Array<AWS.CloudWatchLogs.OutputLogEvent>
   ): Promise<void> => {
     const errorRegexp = /WARN|ERROR|FATAL/;
-    const errors = allLogs.filter(log => errorRegexp.test(log.message));
+    const errors = allLogs.filter(log => errorRegexp.test(log.message)).map(log => log.message);
```

Comment:
Just an observation, I think `.map(log => log.message)` could be just `.map(_.message)` but no need to change for this PR.